packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -1602,3 +1602,160 @@ export class Addr extends Op {
     stack.push(addr.publicKey);
   }
 }
+
+// get balance for the requested account specified
+// by Txn.Accounts[A] in microalgos. A is specified as an account
+// index in the Accounts field of the ApplicationCall transaction,
+// zero index means the sender
+export class Balance extends Op {
+  readonly interpreter: Interpreter;
+
+  /**
+   * @param args words list extracted from line: {}
```

Comment:
please be more precise in the documentation and describe what are the expected arguments in `args`

Reply:
Please update all other constructors documentation in this PR as well.

Reply:
I left it blank because `balance` expect 0 arguments.

Reply:
updated all the other opcodes in parser support for branch PR